src/server/services/recipesService.ts

```diff
@@ -112,6 +112,7 @@ export async function getRecipe(ctx: Context, recipeId: string) {
       nationalities: true,
       mealTypes: true,
```

Comment:
Currently, the many-to-many relations (mealTypes, nationalities, cookingMethods) are return id's instead of the values. For example, it's returning the id instead of Canada for nationalities.